pkg/operator/ceph/csi/spec.go

```diff
@@ -523,6 +523,7 @@ func deleteCSIDriverResources(
 }
 
 func applyCephClusterNetworkConfig(objectMeta *metav1.ObjectMeta, rookclientset rookclient.Interface) (bool, error) {
+	var isMultusApplied bool
 	cephClusters, err := rookclientset.CephV1().CephClusters(objectMeta.Namespace).List(metav1.ListOptions{})
```

Comment:
not related to this PR, but still a  question `objectMeta.Namespace` is the namespace of the operator as csi pods will start in rook operator namespace. will this check passes if we have ceph clusters in different namespaces?

Reply:
@rohan47 is working on a fix for this.

Reply:
So if the CephCluster is not in the operator namespace this won't work.